tools/pythonpkg/tests/fast/arrow/test_nested_arrow.py

```diff
@@ -116,17 +116,11 @@ def test_map_roundtrip(self,duckdb_cursor):
             return
         compare_results("SELECT a from (select MAP(LIST_VALUE(1, 2, 3, 4),LIST_VALUE(10, 9, 8, 7)) as a) as t")
         
-        with pytest.raises(Exception):
-            compare_results("SELECT a from (select MAP(LIST_VALUE(1, 2, 3, 4,2, NULL),LIST_VALUE(10, 9, 8, 7,11,42)) as a) as t")
+        compare_results("SELECT a from (select MAP(LIST_VALUE(1, 2, 3, 4),LIST_VALUE(10, 9, 8, 7)) as a) as t")
         
         compare_results("SELECT a from (select MAP(LIST_VALUE(),LIST_VALUE()) as a) as t")
-        compare_results("SELECT a from (select MAP(LIST_VALUE('Jon Lajoie', 'Backstreet Boys', 'Tenacious D','Jon Lajoie' ),LIST_VALUE(10,9,10,11)) as a) as t")
-        with pytest.raises(Exception):
-            compare_results("SELECT a from (select MAP(LIST_VALUE('Jon Lajoie', NULL, 'Tenacious D',NULL,NULL ),LIST_VALUE(10,9,10,11,13)) as a) as t")
-        with pytest.raises(Exception):
-            compare_results("SELECT a from (select MAP(LIST_VALUE(NULL, NULL, NULL,NULL,NULL ),LIST_VALUE(10,9,10,11,13)) as a) as t")
-        with pytest.raises(Exception):
-            compare_results("SELECT a from (select MAP(LIST_VALUE(NULL, NULL, NULL,NULL,NULL ),LIST_VALUE(NULL, NULL, NULL,NULL,NULL )) as a) as t")
```

Comment:
Isn't the idea of these tests to check that null keys throw an error? Or both Arrow and DuckDB accept null keys ?

Reply:
Hmm fair point, I figured it was testing that Arrow throws an error on this import, but since DuckDB already doesn't accept null keys, this test was obsolete